index.html

```diff
@@ -1169,19 +1169,66 @@ <h2>Relative DID URLs</h2>
 
   </section>
 
-  <section class="informative">
+  <section>
     <h1>Data Model</h1>
-
+    <p>
+This specification defines an abstract data model for <a>DID documents</a> so it
+is independent of any specific representation. In this section we provide a
+high-level description of the data model, a set of requirements for
+representations, and a set of requirements for extensibility.
+    </p>
     <section>
       <h2>Definition</h2>
+      <p>
+A <a>DID document</a> consists of a collection of name-value pairs which will be
+referred to as properties in this document. The definitions of each of these
+properties are specified in section <a href="#core-properties"></a>. Specific
+representations are defined in section <a href="#core-representations"></a>.
+      </p>
     </section>
-
     <section>
-      <h2>Extensibility</h2>
+      <h2>Representations</h2>
+      <p>
+Following are the requirements for conformant representations.
+      </p>
+      <ol>
+        <li>
+A representation MUST define an unambiguous encoding and decoding of all
+properties defined in this specification such that the value of any property
+may be losslessly converted into another representation.
+        </li>
+        <li>
+The representation MUST have an unambiguous MIME type.
+        </li>
+        <li>
+The representation MUST define fragment processing rules for its MIME type that
+are conformant with this specification.
+        </li>
+      </ol>
     </section>
-
     <section>
-      <h2>Representation Requirements</h2>
+      <h2>Extensibility</h2>
+      <p>
+The data model supports two types of extensibility.
+      </p>
+      <ol>
+        <li>
+Registry-based extensions are RECOMMENDED for maximum interoperability. The use
+of the registry for properties is the only guaranteed method for two independent
+implementations to be able to work together. Registry-based extensions MUST be
+losslessly convertible between all conformant representations. The official W3C
+extension registry is defined by the DID Core Registry specification.
+        </li>
+      <li>
+Decentralized extensibility mechanisms MAY be defined by the specification for a
+conformant representation. Decentralized extension mechanisms SHOULD be defined
+such that any conformant extension may be losslessly converted into any other
+conformant representation. However, it's always possible for two specific
```

Comment:
> However ... representations internally

Doesn't feel like it fits with the rest of the normative language. It is also not necessarily tied to "decentralized extensibility mechanisms" - implementations that use the registry sometimes could also choose to make out-of-band agreements for certain extensions. Perhaps this sentence can be de-emphasised in a NOTE box? With the caveat about breaking interop/being non-conformant, something like:

> [NOTE] It's always possible for two specific implementations to agree out-of-band to use a common extension property or a new representation that is not recorded in the [[DID Core Registries]]. It is worth noting that these extensions are not considered conformant to the DID Core specification, and interoperability between implementations is less reliable.

Reply:
Made this change in commit [fdacc41](https://github.com/w3c/did-core/pull/241/commits/fdacc41e349e2a25352fc9ebb6179ee5ea973db1). You are quite right; I turned that language into a Note.